app/src/main/java/ca/bc/gov/vaxcheck/ui/onboarding/OnBoardingFragment.kt

```diff
@@ -53,9 +53,10 @@ class OnBoardingFragment : Fragment(R.layout.fragment_onboarding) {
             })
 
         binding.txtPrivacyPolicy.setSpannableLink {
-            val uri = Uri.parse("https://www2.gov.bc.ca/gov/content/covid-19/vaccine/proof/businesses#app-privacy-policy")
-            val intent = Intent(Intent.ACTION_VIEW, uri)
-            requireContext().startActivity(intent)
+            val uri = Uri.parse(getString(R.string.url_privacy_policy))
+            val action = OnBoardingFragmentDirections
```

Comment:
No need to pass URL from this screen as on that screen we can get it from strings.xml

Reply:
changes done.